app/assets/javascripts/application.js

```diff
@@ -18,6 +18,7 @@
 //= require components/autocomplete.js
 //= require components/image-cropper.js
 //= require components/input-length-suggester.js
+//= require components/gtm-form-listener.js
```

Comment:
I'd suggest this should be in the modules directory of JS since it isn't actually a component

Reply:
Fixed in b955ad0fc27b661e8da6b97adbf4a1e88e58a07c